package-system/pyside2/build_linux.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+# Get the python executable from the package dependency
+LOCAL_PYTHON3_BIN=$TEMP_FOLDER/python-3.10.5-rev2-linux/python/bin/python3
+
+SCRIPT_PATH=`dirname $0`
+
+
+if [ ! -f "$LOCAL_PYTHON3_BIN" ]
+then
+    echo "Missing 3P dependency of python-3.10.5 $LOCAL_PYTHON3_BIN"
+    echo "You will need to build the 3P version of python under the '`readlink -f ../python`'' folder"
+    echo "with the following command:"
+    echo
+    echo "python3 build_package_image.py"
+    echo
+    exit 1
+fi
```

Comment:
Is this still the case?  feels like the package downloader would have sorted this out.

Reply:
This is due to the fact LOCAL_PYTHON3_BIN is hard-coded above, didn't want to add more to the script unless we have more cases where we use these package dependencies directly in the script